lib/utils.nix

```diff
@@ -2,33 +2,28 @@
 
 {
   config.lib.stylix = {
-    # get the opacity values as strings in hexadecimal, integer and floating point values for the four opacity options
-    desktopOpacity-hex = lib.toHexString ((((builtins.ceil (config.stylix.opacity.desktop * 100)) * 255) / 100));
-    desktopOpacity-int = builtins.toString (builtins.ceil (config.stylix.opacity.desktop * 100));
-    desktopOpacity-float = builtins.toString config.stylix.opacity.desktop;
+    # get the opacity values as strings in hexadecimal, integer and floating point values
 
+    opacityToHex = opacity: lib.toHexString (
+      ((builtins.ceil (opacity * 100)) * 255 / 100)
```

Comment:
Should we simplify this function?

I.e. `builtins.ceil (opacity * 255)`

Reply:
```suggestion
      builtins.ceil (opacity * 255)
    )
```